### ISSUE TITLE
Switching to InnoDB modes causes to crash with MySQL 8.0.3

### ISSUE THREAD

**Issue by asayamakk** [innotop/innotop#162]
Hi, I was trying to use innotop with MySQL 8.0.3, and I had faced with crashes.

## to reproduce

- run innotop with MySQL 8.0.3
- press any keys for switching to InnoDB modes,such as D for InnoDB Deadlocks
- innotop will crash with error ` Use of uninitialized value $text in pattern match (m//) at /usr/bin/innotop line 620.`

### keys and modes which will lead to crash.
- B  InnoDB Buffers
- D  InnoDB Deadlocks
- F  InnoDB FK Err
- I  InnoDB I/O Info
- L  Locks
- R  InnoDB Row Ops
- S  Variables & Status
- T  InnoDB Txns

## versions

```bash
$ mysqld --version
# /usr/local/mysql803rc/bin/mysqld  Ver 8.0.3-rc-log for linux-glibc2.12 on x86_64 (MySQL Community Server (GPL))
```

```bash
$ innotop --version
# innotop  Ver 1.11.4
```

```bash
cat /etc/redhat-release
# CentOS Linux release 7.2.1511 (Core)
```

Please let me know if you need further information.

**Comment by metaphor** [comment]
any updates?

**Comment by yoku0825** [comment]
I sent a Pull-Request #167 .

**Comment by lefred** [comment]
fixed in PR #167
